x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_menu.tsx

```diff
@@ -43,33 +56,53 @@ export const ActionTypeMenu = ({ onActionTypeChange, actionTypes }: Props) => {
   }, []);
 
   const registeredActionTypes = Object.entries(actionTypesIndex ?? [])
-    .filter(([index]) => actionTypeRegistry.has(index))
-    .map(([index, actionType]) => {
-      const actionTypeModel = actionTypeRegistry.get(index);
+    .filter(([id, details]) => actionTypeRegistry.has(id) && details.enabledInConfig === true)
+    .map(([id, actionType]) => {
+      const actionTypeModel = actionTypeRegistry.get(id);
       return {
         iconClass: actionTypeModel ? actionTypeModel.iconClass : '',
         selectMessage: actionTypeModel ? actionTypeModel.selectMessage : '',
         actionType,
         name: actionType.name,
-        typeName: index.replace('.', ''),
+        typeName: id.replace('.', ''),
```

Comment:
Yeah, @YulNaumenko may know more why? I'm guessing `.` caused issues in something like a generated className or something?

Reply:
Was wondering if those dot-prefixed names were going to end up causing issues with css selectors :-) !

We should probably come up with a better scheme here, though not sure what it might be.  Replacing `.` with >something< other than `''` seems like it would make sense, but not sure what that would be.  Seems pretty low-risk in any case. 

If that sounds right, suggest we open a tech debt issue to fix later ...

Reply:
Agreed, created issue here: #60424.